kafka_consumer/tests/python_client/test_unit.py

```diff
@@ -178,7 +178,7 @@ def test_tls_config_legacy(extra_config, expected_http_kwargs, check, kafka_inst
         ),
         pytest.param(
             {'kafka_connect_str': 'invalid'},
-            pytest.raises(Exception),
+            does_not_raise(),  # pytest.raises(Exception) why is this different from kafka-python?
```

Comment:
To reviewer: The reason this is different is because the behavior is a little deceptive. We originally wanted this test to check that an invalid `kafka_connect_str` would result in consumer offsets not being collected. 

However, consumer offset failures [do not raise any exceptions](https://github.com/DataDog/integrations-core/blob/9a6a7b989eea0ab8f25e1a94f8433ac21034e34c/kafka_consumer/datadog_checks/kafka_consumer/new_kafka_consumer.py#L70), so actually this test's exception is raised later during the broker offset metric collection should actually check `There was a problem collecting the highwater mark offsets.` exists in the logs. 

Since this PR doesn't implement the broker offset for Confluent library, I will let this continue to error out for now. We will determine if it makes sense to comment out the old tests temporarily as we implement parts of the check.